var/spack/repos/builtin/packages/libtheora/package.py

```diff
@@ -43,12 +46,23 @@ class Libtheora(AutotoolsPackage, MSBuildPackage):
         "msbuild", "autotools", default="autotools" if sys.platform != "win32" else "msbuild"
     )
 
-    patch("exit-prior-to-running-configure.patch", when="@1.1.1")
```

Comment:
If we don't use the patch anymore, we should also remove it from the repository.

Reply:
Thank you, should have thought about that.